flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -108,6 +109,11 @@
 
 	private final ResultStore resultStore;
 
+	// insert into sql match pattern
+	private static final int DEFAULT_PATTERN_FLAGS = Pattern.CASE_INSENSITIVE | Pattern.DOTALL;
+	public static final Pattern INSERT_INTO_SQL_PATTERN = Pattern.compile("(INSERT\\s+INTO.*)",
+			DEFAULT_PATTERN_FLAGS);
+
```

Comment:
We can inline this `DEFAULT_PATTERN_FLAGS ` directly to avoid introduce a new variable.